pkg/istio-agent/agent.go

```diff
@@ -376,19 +377,18 @@ func (a *Agent) Run(ctx context.Context) (func(), error) {
 		return nil, fmt.Errorf("failed to check SDS socket: %v", err)
 	}
 	if socketExists {
-		log.Infof("Existing workload SDS socket found at %s. Default Istio SDS Server won't be started", configuredAgentSocketPath)
-	} else {
+		log.Infof("Existing workload SDS socket found at %s. Default Istio SDS Server will only serve files", configuredAgentSocketPath)
```

Comment:
Update the [above comments](https://github.com/istio/istio/pull/54725/files#diff-31c475bbc33537cd99e6c4e87b7e33dd04dd1a14b3135bcc04d864df3338b374L350-L362) to reflect the changes made in this PR.